demo/load-tests/load-as-es-module.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<!--
+ Copyright 2019 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>EME Encryption Scheme Polyfill - load as ES module</title>
+    <script type="module" src="../../dist/eme-encryption-scheme-polyfill.js"></script>
+    <script>
+      async function testLoader() {
```

Comment:
Don't you get errors from having `async` with no `await`?

Reply:
Not from the browser, no.  It's perfectly legal.  But eslint or Closure might complain with certain settings.